index.html

```diff
@@ -1,14 +1,59 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="ru">
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="pages/index.css">
     <title>Pink App</title>
   </head>
   <body class="page">
-    <div class="page__container">
-      <p>The repository is created with the purpose of learning adaptive web design and team collaboration using Git and GitHub.</p>
+    <div class="page__container">    
+        <header class="header">
+            <picture>
+                <source srcset="images/header-logotype-white-mobile.svg" media="(max-width:320px)">
+                <source srcset="images/header-logotype-white-tablet.svg" media="(max-width:660px)">
+                <img src="images/header-logotype-white-desktop.svg" alt="очки и текст pink" class="logo">
+            </picture>
+                <nav>
+                    <ul class="header__nav-list">
+                        <li>
+                          <a href="" class="header__link-current">Главная</a>
+                        </li>
+                        <li>
+                          <a href="">Фотографии</a>
+                        </li>
+                        <li>
+                          <a href="">Конкурс</a>
+                        </li>
+                        <li>
+                          <a href="">HTML Academy</a>
+                        </li>
+                    </ul>
+                    <button class="header__menu-burger" aria-label="открыть меню"><img src="images/header-menu-burger.png" alt="Меню бургер" ></button>
+                </nav>
+            </header>
+            <section class="intro">
+              <section class="hero">
+                <h1 class="visually-hidden">Pink App</h1>
+                  <h2 class="hero__title">Взгляните на жизнь иначе!</h2>
+                  <div class="hero__info">
+                    <button href="" class="hero__button">Скачать приложение</button>
```

Comment:
A `<button>` tag doesn't have `href` attribute. It is for anchor or link elements. Could you please remove `href` attribute.

Reply:
Done✅